src/js/subtraction/querys.ts

```diff
@@ -1,9 +1,10 @@
 import { useQuery } from "react-query";
-import { findSubtractions } from "./api";
-import { SubtractionSearchResult } from "./types";
+import { fetchSubtractionShortlist, findSubtractions } from "./api";
+import { SubtractionSearchResult, SubtractionShortlist } from "./types";
 
 export const subtractionQueryKeys = {
     list: (filters: Array<string | number | boolean>) => ["subtraction", "list", ...filters] as const,
+    shortlist: () => ["subtraction", "shortlist"] as const,
```

Comment:
```suggestion
    shortlist: () => ["subtraction", "list", "short"] as const,
```
I have been moving all lists over to being some variant of `["subtraction", "list", "OtherKey"]` so that invalidation targeting lists can simply invalidate `Thing.List` to invalidate all the lists. For now we are using all instead, but I am hoping to make it more selective down the line.